docs/api.md

```diff
@@ -1,3720 +1,4067 @@
-
-## Functions
-
-<dl>
-<dt><a href="#getNextLink">getNextLink(response)</a> ⇒</dt>
-<dd><p>getNextLink</p>
-</dd>
-</dl>
-
-<a name="Account"></a>
-
-## Account
-**Kind**: global class  
-
-* [Account](#Account)
-    * [new Account()](#new_Account_new)
-    * [.get(accountId, callback)](#Account+get) ⇒ <code>[AccountResponse](#AccountResponse)</code>
-    * [.getTransactions(params, callback)](#Account+getTransactions) ⇒ <code>[TransactionListResponse](#TransactionListResponse)</code>
-
-<a name="new_Account_new"></a>
-
-### new Account()
-Account
-
-<a name="Account+get"></a>
-
-### account.get(accountId, callback) ⇒ <code>[AccountResponse](#AccountResponse)</code>
-Gets information about user's account.
-
-**Kind**: instance method of <code>[Account](#Account)</code>  
-**Returns**: <code>[AccountResponse](#AccountResponse)</code> - A promise for the account information  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| accountId | <code>String</code> | The ID of the account to get |
-| callback | <code>function</code> | A callback with the account information |
-
-**Example**  
-```js
-// Promise
-client.Account.get().then(function(info){});
-
-// Callback
-client.Account.get(function(err, info){});
-```
-<a name="Account+getTransactions"></a>
-
-### account.getTransactions(params, callback) ⇒ <code>[TransactionListResponse](#TransactionListResponse)</code>
-Gets a list of transactions from user's account.
-
-**Kind**: instance method of <code>[Account](#Account)</code>  
-**Returns**: <code>[TransactionListResponse](#TransactionListResponse)</code> - A promise for the list of transactions  
-
-| Param | Type | Default | Description |
-| --- | --- | --- | --- |
-| params | <code>Object</code> |  | Query parameters for listing accounts |
-| [params.size] | <code>Number</code> | <code>25</code> | Used for pagination to indicate the size of each page requested for querying a list of transactions. If no value is specified the default value is 25 (maximum value 1000). |
-| [params.maxItems] | <code>Number</code> |  | Limit the number of transactions that will be returned |
-| [params.toDate] | <code>String</code> |  | Return only transactions that are newer than the parameter. |
-| [params.fromDate] | <code>String</code> |  | Return only transactions that are older than the parameter. |
-| [params.type] | <code>String</code> |  | Return only transactions that are this type. |
-| callback | <code>function</code> |  | A callback with the list of transactions |
-
-**Example**  
-```js
-//Promise
-client.Account.getTransactions()
-	.then(function (response) {
-		console.log(response.transactions);
-		if(response.hasNextPage) {
-			return response.getNextPage();
-		}
-		else {
-			return {transactions: []};
-		}
-	})
-	.then(function(response) {
-		console.log(response.transactions);
-	});
-```
-**Example**  
-```js
-//Get transactions filtering by date
-//Promise
-var params = {
-	fromDate: "2013-02-21T13:38:00"
+var util = require("util");
```

Comment:
Whoops, I overwrite api.md! Don't merge this file. – the rest is still applicable 👍

Reply:
Hey @amiller-gh 
I checked the team. Could you check if https://github.com/bandwidth/node-messaging has what you're looking for? It's the newer SDKs, I think.